provider/pkg/gen/schema.go

```diff
@@ -50,7 +50,7 @@ func PulumiSchema() (*schema.PackageSpec, *resources.CloudAPIMetadata, error) {
 	}
 
 	csharpNamespaces := map[string]string{
-		"gcp-native": "GcpNative",
+		"google-native": "GcpNative",
```

Comment:
Should the namespace change as well?

Reply:
@viveklak @mikhailshilkov ?

Reply:
Sorry I hammered this through to fix a build for docs locally.  I will fix this shortly. Thanks for pointer @lblackstone

Reply:
Follow-up here: https://github.com/pulumi/pulumi-google-native/pull/50